dcos_launch/config.py

```diff
@@ -251,7 +274,30 @@ def get_validated_config(user_config: dict, config_dir: str) -> dict:
             'public_agent_list': {'readonly': True},
             'fault_domain_script_filename': {
                 'coerce': 'expand_local_path',
-                'excludes': 'fault_domain_script_contents'}}}}
+                'excludes': 'fault_domain_script_contents'}}},
+    'fault_domain_helper': {
+        'type': 'dict',
```

Comment:
should `'validator': _validate_fault_domain_helper` go here?

Reply:
Yup! Fixed in most recent update